### ISSUE TITLE
Cannot build droplet if it contains a symlink

### ISSUE THREAD

**Issue by oppegard** [cloudfoundry-incubator/lattice#170]
To reproduce: 

```bash
$ git clone 'https://github.com/pivotal-cf-experimental/xray.git'
$ cd xray
$ ln -s index.js index-symlink.js
$ ltc build-droplet xray nodejs -t 4m
Submitted build of xray
08/11 15:24:23.85 [BUILD|0] Creating container
08/11 15:24:24.18 [BUILD|0] Successfully created container
Build failed: Copying into the container failed
```

Logs:
```
[garden-linux|cell-01] [ERROR]   08/11 15:24:24.61 124.1          garden-linux.stream-in.command.failed
                                                                  exit status 2
                                                                  {"argv":["/var/lattice/garden/depot/uc4db1cqk11/bin/nstar","14446","vcap","/tmp/app"],"exit-status":2,"stderr":"tar: index-symlink.js: Cannot create symlink to '': No such file or directory\ntar: Exiting with failure status due to previous errors\n","stdout":"","took":"10.328413ms"}
[garden-linux|cell-01] [ERROR]   08/11 15:24:24.61 4.791          garden-linux.garden-server.stream-in.failed
                                                                  error streaming in: exit status 2. Output: tar: index-symlink.js: Cannot create symlink to '': No such file or directory
tar: Exiting with failure status due to previous errors

                                                                  {"destination":"/tmp/app","handle":"build-droplet-xray","user":"vcap"}
[rep|cell-01]          [ERROR]   08/11 15:24:24.61 4.127.2.1.2    rep.depot-client.run-container.run.action.download-step.stream-in-failed
                                                                  error streaming in: exit status 2. Output: tar: index-symlink.js: Cannot create symlink to '': No such file or directory
tar: Exiting with failure status due to previous errors

                                                                  {"cacheKey":"","destination":"/tmp/app","guid":"build-droplet-xray","to":"/tmp/app","user":"vcap"}
```

I was able to push/stage the same bits with the `cf` CLI; it seems to ignore symlinks altogether.

**Comment by trisberg** [comment]
I keep getting the same error trying to build a simple Spring Boot app.

**Comment by menicosia** [comment]
Yes, sorry. This is 100% a bug. It's fixed in Master and will be included in the next release. Look for it next week!

--
Marco Nicosia

**Comment by trisberg** [comment]
OK, thanks for the feedback, will check back next week.

**Comment by davidwadden** [comment]
This landed with [**v0.3.3**](https://github.com/cloudfoundry-incubator/lattice/releases/tag/v0.3.3), please try it out!

Closing issue.
